node/src/chain_spec.rs

```diff
@@ -197,7 +197,7 @@ pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 		None,
 		Some(serde_json::from_str("{\"tokenDecimals\": 18}").expect("Provided valid json map")),
 		Extensions {
-			relay_chain: "local_testnet".into(),
+			relay_chain: "rococo-local".into(),
```

Comment:
Is this intentional?

Reply:
yes. `local_testnet` was never correct. Here are the valid options https://github.com/paritytech/polkadot/blob/master/cli/src/command.rs#L77-L92